apps/aehttp/test/aehttp_ws_test_utils.erl

```diff
@@ -109,9 +115,12 @@ onconnect(_WSReq, Regs) ->
     inform_registered(WaitingPid, websocket, connected),
     {ok, Regs1}.
 
+
+ondisconnect({error, {400, <<"Bad Request">>}}, Regs) ->
```

Comment:
When does this happen? Worth documenting in protocol repo?

Reply:
When all Websocket acceptors are consumed and the queue for incoming new Websockets is full, we start rejecting new connections. This is the standart `cowboy` message being send to the request. It is not documented in the protocol repo, I'm not sure if it would add more clarity or the opposite.

Reply:
mmm it could surprise some clients so I would consider it worth documenting it